src/ray/common/ray_config_def.h

```diff
@@ -489,7 +489,7 @@ RAY_CONFIG(int64_t, idle_worker_killing_time_threshold_ms, 1000)
 RAY_CONFIG(int64_t, num_workers_soft_limit, -1)
 
 // The interval where metrics are exported in milliseconds.
-RAY_CONFIG(uint64_t, metrics_report_interval_ms, 30000)
+RAY_CONFIG(uint64_t, metrics_report_interval_ms, 5000)
```

Comment:
can we revert this for now? I'd like to set it 5000 after some follow-up items of agent leak issue!

Reply:
Sounds good.